.claude/skills/build/SKILL.md

```
@@ -101,20 +129,34 @@ Before starting, remove any leftover files from a previous build run:
 
 **If BRAINSTORM=true** — resume the agent from Step 0.5:
 - `resume: "<agent-id-from-step-0.5>"`
-- Prompt: `MODE: DISCOVERY\n\nChosen design direction: <CHOSEN_DESIGN>\n\n<clean PRD content>`
-- Tell it to output questions to `tasks/planning-questions.md`
+- Prompt:
+  ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)